diff -Nru MyShogi.org/MyShogi/Model/Dependency/Mono/MonoAPI.cs MyShogi/MyShogi/Model/Dependency/Mono/MonoAPI.cs
--- MyShogi.org/MyShogi/Model/Dependency/Mono/MonoAPI.cs	2020-10-11 22:12:46.760779689 +0900
+++ MyShogi/MyShogi/Model/Dependency/Mono/MonoAPI.cs	2020-10-11 22:13:35.768037995 +0900
@@ -222,19 +222,14 @@
             }
         }
 
-        // htmlファイルをブラウザで開く
-        // (htmlが拡張子関連付けされているアプリで開く)
-        // ※　この実装でLinux環境で動くのかは未確認。[2020/05/10]
-        // 外部エンジン読み込み用の説明ファイル(html)を開くのに用いている。
+        // 外部エンジン読み込み用の説明ファイル(html)をブラウザで開く。
         public static void OpenWithBrowser(string htmlPath)
         {
-            var current = Path.GetDirectoryName(Application.ExecutablePath);
-            var filePath = Path.Combine(current, htmlPath);
-
-            var p = new Process();
-            p.StartInfo.FileName = "file:///" + filePath;
-            p.StartInfo.UseShellExecute = true;
-            p.Start();
+#if MACOS
+                Process.Start("open", htmlPath);
+#elif LINUX
+                Process.Start("sensible-browser", htmlPath);
+#endif
         }
     }
 
