diff -Nru MyShogi.org/MyShogi/View/Win2D/Setting/EngineDefineEditDialog.Designer.cs MyShogi/MyShogi/View/Win2D/Setting/EngineDefineEditDialog.Designer.cs
--- MyShogi.org/MyShogi/View/Win2D/Setting/EngineDefineEditDialog.Designer.cs	2020-10-12 15:32:18.148392076 +0900
+++ MyShogi/MyShogi/View/Win2D/Setting/EngineDefineEditDialog.Designer.cs	2020-10-12 15:34:24.587190650 +0900
@@ -387,8 +387,9 @@
             this.label15.Name = "label15";
             this.label15.Size = new System.Drawing.Size(431, 24);
             this.label15.TabIndex = 1;
-            this.label15.Text = "Step 4. 書き出されたエンジン定義ファイルを思考エンジンのフォルダに移動させて、\r\n　思考エンジンを\"Documents/myshogi-engines/エ" +
-    "ンジン名/\"配下に移動させてください。";
+            this.label15.Text = "Step 4. 書き出されたエンジン定義ファイル\"engine_define.xml\"を\r\n　" + 
+    "思考エンジンのフォルダ\"<MyShogiのインストール先>/engine/" + 
+    "<思考エンジン名>/\"配下に移動させてください。";
             // 
             // label16
             // 
