--- MyShogi.org/MyShogi/View/Win2D/Game/GameScreenControl/GameScreenControlConst.cs	2020-10-21 11:08:46.267572523 +0900
+++ MyShogi/MyShogi/View/Win2D/Game/GameScreenControl/GameScreenControlConst.cs	2020-10-23 15:44:36.085562452 +0900
@@ -66,24 +66,25 @@
         {
             // 普通の駒台の場合
             {
-                // 10(margin)+96(piece_width)+30(margin)+96(piece_width)+28(margin) = 260(駒台のwidth)
-                new Point(/*Piece.PAWN  ,*/  10,280),
-                new Point(/*Piece.LANCE ,*/ 135,190),
-                new Point(/*Piece.KNIGHT,*/  10,190),
-                new Point(/*Piece.SILVER,*/ 135,100),
-                new Point(/*Piece.BISHOP,*/ 135,  5),
-                new Point(/*Piece.ROOK  ,*/  10,  5),
-                new Point(/*Piece.GOLD  ,*/  10,100),
+                // (横,縦)=(260,388)
+                new Point(/*Piece.PAWN  ,*/ 168,260),
+                new Point(/*Piece.LANCE ,*/  82,260),
+                new Point(/*Piece.KNIGHT,*/ -10,260),
+                new Point(/*Piece.SILVER,*/ 131,135),
+                new Point(/*Piece.BISHOP,*/ 135, 10),
+                new Point(/*Piece.ROOK  ,*/  28, 10),
+                new Point(/*Piece.GOLD  ,*/  32,135),
             },
             // 縦長の駒台の場合
             {
-                new Point(/*Piece.PAWN  ,*/ -5,549),
-                new Point(/*Piece.LANCE ,*/ -5,459),
-                new Point(/*Piece.KNIGHT,*/ -5,369),
-                new Point(/*Piece.SILVER,*/ -5,279),
-                new Point(/*Piece.BISHOP,*/ -5, 95),
-                new Point(/*Piece.ROOK  ,*/ -5,  0),
-                new Point(/*Piece.GOLD  ,*/ -5,187),
+                // (横,縦)=(95,633)
+                new Point(/*Piece.PAWN  ,*/ -5,555),
+                new Point(/*Piece.LANCE ,*/ -5,460),
+                new Point(/*Piece.KNIGHT,*/ -5,365),
+                new Point(/*Piece.SILVER,*/ -5,266),
+                new Point(/*Piece.BISHOP,*/ -5, 64),
+                new Point(/*Piece.ROOK  ,*/ -5,-37),
+                new Point(/*Piece.GOLD  ,*/ -5,166),
             }
         };
 
